src/App.vue

```diff
@@ -1318,13 +1285,22 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   /** Fetches authorizations and verifies roles. */
   private async checkAuth (id: string): Promise<any> {
     // NB: will throw if API error
-    const authRoles = await AuthServices.fetchAuthorizations(id)
+    const authorizations = await AuthServices.fetchAuthorizations(id)
+    const authRoles: Array<AuthorizationRoles> = authorizations.roles || []
+
+    if (!Array.isArray(authRoles)) {
+      throw new Error('Invalid auth roles')
+    }
 
-    // verify that array has at least one role
-    // NB: roles array may contain 'view', 'edit', 'staff' or nothing
-    if (!Array.isArray(authRoles) || authRoles.length < 1) {
+    // verify that array has "view" or "staff" roles
+    if (
+      !authRoles.includes(AuthorizationRoles.VIEW) &&
+      !authRoles.includes(AuthorizationRoles.STAFF)
+    ) {
```

Comment:
It appears that all account types have either role "view" or "staff" (plus many others), so the above should be good.